velox/expression/VectorUdfTypeSystem.h

```diff
@@ -203,6 +203,25 @@ struct VectorReader {
     return decoded_.mayHaveNulls();
   }
 
+  // These functions can be used to check if any elements in a given row are
+  // NULL. They are not especially fast, so they should only be used when
+  // necessary, and other options, e.g. calling mayHaveNullsRecursive() on the
+  // vector, have already been exhausted.
+  inline bool containsNull(vector_size_t index) const {
+    return decoded_.isNullAt(index);
+  }
+
+  inline bool containsNull(vector_size_t startIndex, vector_size_t endIndex)
+      const {
```

Comment:
maybe add //TODO or note that this can be made faster if specialized for flat encoding using bit operations.